### ISSUE TITLE
[BottomNavigation] minimal tap area not being enforced

### ISSUE THREAD

**Issue by yarneo** [material-components/material-components-ios#4306]
Currently with bottom navigation we set the tap size of each item based on its size, but when there are a lot of items the tap size becomes smaller and smaller and there is no enforcement for this.

**Comment by romoore** [comment]
Can you provide an example/screenshot of the problem?

**Comment by romoore** [comment]
The Material Guidelines are pretty clear that [no fewer than 3 and no more than 5 tabs should be used](https://material.io/design/components/bottom-navigation.html#usage). We should definitely make sure that tap targets are sufficiently large for 4/5 items in compact layouts:

1. iPad in side-by-side multitasking/portrait
2. iPhone 4s

**Comment by yarneo** [comment]
If the guidelines don't allow more than 5 items we should be good!
![screen shot 2018-05-29 at 3 51 22 pm](https://user-images.githubusercontent.com/4066863/40681859-48737038-6358-11e8-92d7-904e7ea888ba.png)

**Comment by romoore** [comment]
Oh my! Yes, I think an app like that would have bigger  UX issues than the bottom navigation tabs. >_<
